external/build/fugu-showcase.js

```diff
@@ -0,0 +1,176 @@
+  const results = [];
+
+  async function runTasks(tasksIterator) {
+    for (const [index, task] of tasksIterator) {
+      try {
+        results[index] = await task();
+      } catch (error) {
+        // @ts-ignore
+        results[index] = new Error(`Failed with: ${error.message}`);
+      }
+    }
+  }
+
+  const workers = new Array(concurrency).fill(tasks.entries()).map(runTasks);
+  await Promise.allSettled(workers);
+  return results;
+}
+
+const createScreenshots = async (data, overrideType = null) => {
+  // eslint-disable-next-line node/no-unsupported-features/es-syntax
+  const captureWebsite = (await import('capture-website')).default;
+  let items = data.map(item => [item.appURL, item.screenshot]);
+  const length = items.length;
+  if (!overrideType) {
+    items = items.concat(items);
+  }
+  const tasks = items.map(([url, filename], i) => {
+    return () => {
+      SCREENSHOT_OPTIONS.type = overrideType || SCREENSHOT_OPTIONS.type;
+      SCREENSHOT_OPTIONS.darkMode = i >= length;
+      filename = SCREENSHOT_OPTIONS.darkMode
+        ? filename.replace(
+            new RegExp(`(.${SCREENSHOT_OPTIONS.type}$)`),
+            '-dark$1'
+          )
+        : filename;
+      if (!SCREENSHOT_OPTIONS.darkMode) {
+        data[i].screenshot = filename;
+      } else {
+        data[i - length].screenshotDark = filename;
+      }
+      // @ts-ignore
+      return captureWebsite.buffer(url, SCREENSHOT_OPTIONS).then(buffer => {
+        const fileLocation = 'site/en/fugu-showcase';
+        if (!overrideType) {
+          return sharp(buffer)
+            .resize({
+              width: SCREENSHOT_OPTIONS.width / 2,
+              height: SCREENSHOT_OPTIONS.height / 2,
+            })
+            .toBuffer()
+            .then(data => {
+              return fs
+                .writeFile(path.resolve(fileLocation, filename), data)
+                .then(() =>
+                  console.log(`Successfully created \`${filename}\`.`)
+                );
+            })
+            .catch(err => console.error(err));
+        }
+        return fs
+          .writeFile(path.resolve(fileLocation, filename), buffer)
+          .then(() => console.log(`Successfully created \`${filename}\`.`));
+      });
+    };
+  });
+  await limit(tasks, 5);
+};
+
+async function run() {
+  if (process.env.CI) {
+    return; // do nothing, the fallback data will win
+  }
+  // eslint-disable-next-line node/no-unsupported-features/es-syntax
+  const fileNamifyURL = (await import('filenamify-url')).default;
+  try {
+    const response = await fetch(SPREADSHEET_URL);
+    if (!response.ok) {
+      throw new Error(
+        `Could not fetch Fugu entries. Status code was: ${response.status}.`
+      );
+    }
+    const json = await response.json();
+    if (json['errors']) {
+      const error = json['errors'][0];
+      throw new Error(`${error.code}: ${error.message}`);
+    }
+
+    const unavailableServices = [
```

Comment:
Move to `const SKIPPED_SERVICES` at the top of the file, though this could probably be also a flag in the underlying spreadsheet...?